lib/net/pool.js

```diff
@@ -213,7 +222,12 @@ class Pool extends EventEmitter {
     this.headerNext = null;
 
     const tip = this.chain.tip;
-
+    if (this.options.neutrino) {
+      this.headerChain.push(new HeaderEntry(tip.hash, tip.height));
+      this.cfHeaderChain = new List();
+      this.cfHeaderChain.push(new CFHeaderEntry(consensus.ZERO_HASH, 0));
+      return;
+    }
```

Comment:
This makes me think we need a second function `resetCFChain()` or something. All three node varieties can call this existing `resetChain()` but only neutrino will reset the CFChain